common/core/desktop/tests/unit/kmnkbd/test_kmx_xstring.cpp

```diff
@@ -17,6 +17,19 @@
 using namespace km::kbp::kmx;
 using namespace std;
 
+#define U_UC_SENTINEL u"\uFFFF"
+#define U_UC_SURROGATE u"\U0001F609"
```

Comment:
I don't think I'd use this name, given it's just a random character!

And ... now I see:

> I also added a define for the surrogate pair, but I'm not sure if that makes things clearer or hides the fact that it's two bytes.

Yeah, I think it's better to give it a different name:

```suggestion
#define U_1F609_WINKING_FACE u"\U0001F609"
```

That may be overkill but it's clear and self-documenting!

Reply:
That's an idea to put the code point in the name. Done!